dynamiqs/mesolve/rouchon.py

```diff
@@ -19,6 +19,34 @@ def inv_kraus_matmul(A: Tensor, B: Tensor, upper: bool) -> Tensor:
     return B
 
 
+def batched_mult_left(x, y):
+    r"""Computes the batched matrix product of two tensors where the left tensor
+    has one more dimension than the right one.
+     Args:
+         x: Tensor of shape (N, ..., n, k)
+         y: Tensor of shape (..., k, m)
+
+     Returns:
+        The batched product of `x @ y`
+         `(N, ..., n, m)`.
+    """
```

Comment:
Pls fix extra spaces, missing newlines and missing end points (taking as reference `dynamiqs.utils.utils` docstrings for example).